client/tests/e2e/specs/sidebar_persistence.spec.ts

```diff
@@ -0,0 +1,65 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { expect } from '@playwright/test';
+import { MsPage, msTest } from '@tests/e2e/helpers';
+
+async function isSidebarVisible(page: MsPage): Promise<boolean> {
+  const sidebar = page.locator('.sidebar');
+  return await sidebar.isVisible();
+}
+
+async function toggleSidebar(page: MsPage): Promise<void> {
+  // Look for toggle button in standard locations - simplified approach
+  const toggleButton = page.locator('#trigger-toggle-menu-button').first();
+
+  // Wait for button to be available and click it
+  await toggleButton.waitFor({ state: 'visible', timeout: 10000 });
```

Comment:
Why not just `await expect(toggleButton).toBeVisible()`? Also 10s timeout seems way overkill. As a general rule: if you have to rely on timeouts, the test is going to be flaky.